build.sh

```diff
@@ -189,18 +275,23 @@ function parseArgs() {
     done
 }
 
+function checkJava() {
```

Comment:
This is a nice touch. It's probably not within the scope of the original issue here, but it would be nice to eventually have a check to make sure it's using JDK 11 for building jmc, otherwise currently the build script fails without displaying any helpful information.

Reply:
I feel similar, that it can be outside of the scope, I was already thinking about it. I think that the current the script is meant to be executed from the similar command-line and with not mutable JDK_HOME